mlx/backend/common/jit_compiler.cpp

```diff
@@ -133,7 +133,7 @@ std::string JitCompiler::exec(const std::string& cmd) {
   if (status == -1) {
     throw std::runtime_error("pclose() failed.");
   }
-#ifdef _MSC_VER
+#ifdef WIN32
```

Comment:
Should use `_WIN32`, for MSVC the `WIN32` macro is defined by `windows.h` which is not included everywhere.

Reply:
Fixed